addons/binding/org.openhab.binding.synopanalyzer/src/main/java/org/openhab/binding/synopanalyzer/handler/SynopAnalyzerHandler.java

```diff
@@ -61,6 +62,7 @@
 
     public SynopAnalyzerHandler(Thing thing) {
         super(thing);
+        SYNOP_DATE_FORMAT.setTimeZone(TimeZone.getTimeZone("UTC"));
```

Comment:
Okay so the point of this change is that the synop end point wants to receive the time in UTC? Is that correct or is there another conversion which I am missing?